Develop

crzypatchwork opened the pull request.

violetforest committed:
reset to bare bone

violetforest committed:
fetch latest of first fren is list

violetforest committed:
make it its own page

violetforest committed:
Merge remote-tracking branch 'upstream/develop' into feature/frens

# Conflicts:
#	src/styles/main.scss

violetforest committed:
route to friends page

violetforest committed:
friends/walletaddress

violetforest committed:
differentiate friend and manage assets page

violetforest committed:
feed without scroll

violetforest committed:
add as feed item component

violetforest committed:
add creator address to props

violetforest committed:
remove unnecessary code

andrevenancio committed:
Merge pull request #889 from violetforest/feature/frens

Feature/frens

andrevenancio committed:
Merge branch 'main' into develop

andrevenancio committed:
Merge branch 'main' into develop

pichiste committed:
Fix for merge/revert issue on mint page

pichiste committed:
re-enable mintJ

pichiste committed:
Merge remote-tracking branch 'upstream/develop' into develop

pichiste committed:
updated ipfs util

andrevenancio committed:
Merge branch 'main' into develop

andrevenancio committed:
reverting css

crzypatchwork committed:
Merge pull request #896 from pichiste/develop

Re-implementing external thumbnail generation tool

violetforest committed:
get amoutn of hdao and display on creator profile, floor it

crzypatchwork committed:
Merge pull request #910 from violetforest/feature/hdao-display

display hdao amount on profile

violetforest committed:
clean up code from last PR

violetforest committed:
update with main

andrevenancio committed:
Merge pull request #915 from violetforest/feature/hdao-display-2

Update: Feature/hdao-display

andrevenancio commented:
This will likely break develop like it did a couple of days ago when the minting got broken.

Basically @pichiste has a fancy version on develop that is better than the version on main for the mint. (The resize of the videos, and images for displayUri).

But you never told me to merge it @crzypatchwork so I never did. But basically it asks users to go to an external site to generate thumbnails, it allows the users to download the thumbnails and then they need to upload the zip file they got from that external site.

That version is better, but its still only working on develop.

Would be nice if you could test it, and let us know if you want it on main. I can merge it after. Its quite different on the `mint` and on the `ipfs.js` file that mint calls

violetforest committed:
dont show your objkts in feed, null check

andrevenancio committed:
Merge pull request #917 from violetforest/feature/frens

dont show your objkts in feed, message if you haven't collected anything

Commit:
image when minting audio

crzypatchwork committed:
Merge pull request #949 from lauzaki/feature/mint_audio_img

image in preview when minting audio

crzypatchwork committed:
Revert "image in preview when minting audio"

crzypatchwork committed:
Merge pull request #967 from hicetnunc2000/revert-949-feature/mint_audio_img

Revert "image in preview when minting audio"

crzypatchwork committed:
Revert "Revert "image in preview when minting audio""

crzypatchwork committed:
Merge pull request #969 from hicetnunc2000/revert-967-revert-949-feature/mint_audio_img

Revert "Revert "image in preview when minting audio""

csb324 committed:
Fix the 'hic et nunc - hic et nunc' titles

crzypatchwork committed:
Merge pull request #988 from csb324/feature/title-fix

Page title fix!

sbihel committed:
Add Discord integration to profiles

Uses tzprofiles.
Add a button to copy the Discord handle of the profile.

Co-authored-by: Gregório Granado Magalhães <greg.magalhaes@gmail.com>

w4ll3 committed:
Removed button click animation

w4ll3 committed:
Updated handle/text size to match each other

w4ll3 committed:
Fix the arrow

crzypatchwork committed:
Merge pull request #1019 from spruceid/develop

Feature - Add Discord Verification From Tezos Profiles

danzen committed:
Create README.md

danzen committed:
Added ZIM Interactive NFT template files

ZIM is a JavaScript Canvas Framework at https://zimjs.com with interactive NFT information at https://zimjs.com/nft.  Code Creativity!

crzypatchwork committed:
Merge pull request #1110 from danzen/Adding-ZIM-Interactive-NFT-Template

Adding ZIM Interactive NFT Template
